Gruntfile.js

```diff
@@ -338,6 +362,9 @@ module.exports = function(grunt) {
     'uglify:sourcemapin_sources',
     'uglify:expression_json',
     'uglify:expression_js',
+    'uglify:quotes_single',
+    'uglify:quotes_double',
+    'uglify:quotes_original',
```

Comment:
@vladikoff: how about we make this just `uglify` instead of specifying each subtarget?